MPC/config/googlemock.mpb

```diff
@@ -1,5 +1,8 @@
 project : googletest {
-  includes += $(DDS_ROOT)/tests/googletest/googlemock/include
-  libpaths += $(DDS_ROOT)/tests/googletest/build/googlemock/$(Configuration)
+
+  // For backwards-compatibility.
```

Comment:
This isn't going to make sense to an external audience.
It could be "for compatibility with tests/gtest_setup.txt instructions" or "for compatibility with build tree (not installed) gtest".
Same thing in googletest.mpb.

Reply:
Agreed. I'll get the change in!